chrome.py

```diff
@@ -21,28 +27,50 @@ def __init__(self, error):
 def alfred_error(error):
     alfred.write(alfred.xml([ErrorItem(error)]))
 
-def copy_history(profile):
-    if os.path.isfile(HISTORY_CACHE) and time.time() - os.path.getmtime(HISTORY_CACHE) < HISTORY_CACHE_EXPIRY:
-        return HISTORY_CACHE
+def copy_db(name, profile):
+    cache = os.path.join(alfred.work(True), name)
+    if os.path.isfile(cache) and time.time() - os.path.getmtime(cache) < CACHE_EXPIRY:
+        return cache
 
-    history_file = os.path.join(os.path.expanduser(profile), HISTORY_DB)
+    file = os.path.join(os.path.expanduser(profile), name)
```

Comment:
python has a `file` built-in? WTF. anyway, learned something new. and just checked, looks like reason returned and it has been removed in Python3:

```
$ python3
Python 3.4.2 (default, Jan 29 2015, 06:34:22)
[GCC 4.2.1 Compatible Apple LLVM 6.0 (clang-600.0.56)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> file()
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
NameError: name 'file' is not defined
```